Beagbarreto/employee-info
# Feat/routes

## WHAT THIS PR DOES
Implementation of router, switches from Form to List component, rendering inside App component.
Single page rendering.

## HOW TO TEST THIS PR
When you click on the Form and List button-texts you should be able to see the content from the different components render.

## ADDITIONAL INFORMATION
None

## WHAT DOES "DONE" MEAN?

This PR has been:
- functionality has been verified against screen checklist
- commented / documented

## COMMIT HISTORY
